run_project_tests.py

```diff
@@ -911,8 +911,8 @@ def load_test_json(t: TestDef, stdout_mandatory: bool, skip_category: bool = Fal
             exclude = False
             opt_tuple = [(x[0], x[1]) for x in i]
             for j in matrix['exclude']:
-                ex_list = [(k, v) for k, v in j.items()]
-                if all([x in opt_tuple for x in ex_list]):
+                ex_list = list(j.items())
+                if all(x in opt_tuple for x in ex_list):
```

Comment:
It doesn't look like `ex_list` is used for anything other than this `all()` check below. Maybe we can just fold this into:
`if all(x in opt_tuple for x in j.items())`

Reply:
```suggestion
                if all(x in opt_tuple for x in j.items()):
```